deployments/examples/ocis_multi/docker-compose.yml

```
@@ -0,0 +1,230 @@
+---
+version: "3.7"
+
+services:
+  traefik:
+    image: traefik:v2.9.1
```

```suggestion
    image: traefik:v3.6.2
```
We will use that tag for the `ocis_full` example too, just preparing the PR for it. Tested on hetzner, with ocis_full, works. We should keep it in-sync.